parts/k8s/windowskubeletfunc.ps1

```diff
@@ -204,14 +204,14 @@ New-InfraContainer {
 
     if ($pauseImageVersions -icontains $windowsVersion) {

```

Comment:
I think this whole section would be skipped and the custom pause image wouldn't be pulled.  The `$global:AlwaysPullWindowsPauseImage` needs be added up here for the image to be pulled at provisioning time

Reply:
I think that it will not be skipped since we only added PauseImage for those supported kernel versions.

Reply:
I tested this PR with aks-engine and confirmed that the PauseImage will be pulled when `$global:AlwaysPullWindowsPauseImage` is true.

Reply:
Yes I had it backwards.  Though I think in the case of a insider base image where you provide a custom pause image, it would not get pulled which would lead to some confusion I think.